src/Sulu/Component/CustomUrl/Generator/Generator.php

```diff
@@ -48,6 +48,10 @@ public function generate($baseDomain, $domainParts, Localization $locale = null)
         }
 
         foreach ($domainParts['suffix'] as $suffix) {
+            if (empty($suffix)) {
+                continue;
+            }
```

Comment:
because you could "push" empty parts into the api which is quite bad and should not work.